OrbitLinuxTracing/TracerThread.cpp

```diff
@@ -731,13 +731,20 @@ void TracerThread::Run(const std::shared_ptr<std::atomic<bool>>& exit_requested)
   }
 
   // Close the file descriptors.
-  {
+  // Do it in a separate dedicated thread as this can take a long time
+  // with a lot of uprobes and uretprobes.
+  std::thread([tracing_fds = tracing_fds_] {
+    pthread_setname_np(pthread_self(), "close(fd)s");
     ORBIT_SCOPE_WITH_COLOR("Closing file descriptors", orbit::Color::kRed);
-    for (int fd : tracing_fds_) {
+    uint64_t fd_close_begin_ns = MonotonicTimestampNs();
```

Comment:
Why the additional timing if we have the introspection scope already? Do you always want to have this in the logs?

Reply:
I want it in the logs.

Reply:
Don't we have a similar marco for the logs?

Reply:
Do we? And with a custom message like done here?

Reply:
We have SCOPE_TIMER_LOG but it's in OrbitCore, it would be cool to have it in OrbitBase